infoschema/tables.go

```diff
@@ -1997,7 +2011,81 @@ func dataForTiDBClusterInfo(ctx sessionctx.Context) ([][]types.Datum, error) {
 	return rows, nil
 }
 
-func dataForClusterConfig(ctx sessionctx.Context) ([][]types.Datum, error) {
+func dataForClusterLoadInfo(ctx sessionctx.Context) ([][]types.Datum, error) {
+	serversInfo, err := getClusterServerInfo(ctx)
+	if err != nil {
+		return nil, err
+	}
+	ipMap := make(map[string]struct{}, len(serversInfo))
+	rows := make([][]types.Datum, 0, len(serversInfo)*10)
+	for _, srv := range serversInfo {
+		// TODO: remove this after PD/TiKV support diagnostic grpc service.
+		if srv.tp == "pd" || srv.tp == "tikv" {
+			continue
+		}
+		addr := srv.statusAddr
+		if _, ok := ipMap[addr]; ok {
```

Comment:
I think there is no possibility to use the same address between different cluster nodes.

Reply:
It's a mistake, it should use ip here.